mod_reforged/hooks/skills/perks/perk_nine_lives.nut

```diff
@@ -0,0 +1,40 @@
+::mods_hookExactClass("skills/perks/perk_nine_lives", function(o) {
+    local create = o.create;
+	o.create = function()
+	{
+        create();
+
+        // New Variables to control and tweak the behavior of this perk
+        this.m.MinHP <- 11;
+        this.m.MaxHP <- 15;
+
+        // This makes the perk now always show by default on top if characters while it is not procced yet
+		this.m.IconMini = "perk_07_mini";
+		this.m.Overlay = "perk_07";
+        this.m.Type = (this.m.Type | this.Const.SkillType.StatusEffect);    // Therefor it needs the StatusEffect type
```

Comment:
No need for parantheses.

Reply:
I agree but I prefer it for readability

Reply:
I can _maybe_ see what you mean, but I don't find it convincing enough. I'll let @Enduriel say the final word on this.

Reply:
Eventually I want Enduriels final word on this because we will encounter those situations in the future.
But in this case it doesn't matter because I will replace the line with an `addSkillType` as soon as you guys released the custom MSU version with that feature

Reply:
Ok yes that'll be fine.